src/args.rs

```diff
@@ -54,21 +54,11 @@ where
     // parse scheduled channel arguments: pairs of (addr, amount)
     let mut channels = Vec::with_capacity(args.len() / 2);
 
-    // the remaining single argument is the wallet amount in satoshis (if any)
-    let wallet_amount = loop {
-        match (args.next(), args.next()) {
-            // we have a pair of arguments, interpret it as a scheduled channel (p2p addr, amount)
-            (Some(addr), Some(amount)) => {
-                channels.push(ScheduledChannel::from_args(addr, amount)?);
-            }
-            // if there is a remaining single argument, it is the wallet amount
-            (Some(amount), None) =>
-                break bitcoin::Amount::from_str_in(amount, bitcoin::Denomination::Satoshi)
-                    .map_err(ArgError::InvalidWalletAmount)?,
-            // if there is no remaining single argument, the wallet amount is 0
-            _ => break bitcoin::Amount::ZERO,
-        }
-    };
+    while let (Some(addr), Some(amount)) = (args.next(), args.next()) {
```

Comment:
It may be useful for simple testing?

Reply:
I think as we're adding the ability to get inbound and outbound the decision is really

1. keep it (as-is)
2. kill it
3. keep upgrading it with the ui

And I'm partial to killing it, because all of the users I know are using the ui, and we're testing with the library code

Reply:
Oh, yeah, 3 looks like maintenance burden and integration tests can simply call the API anyway.

Do you have some doc on in-bound channels? I was wondering about them but concluded they can not be done securely with LND API at that time. But maybe the API did change?

Reply:
The idea for inbound is to rent a channel from an lsp with a payment included in the bip21. Read more here: https://github.com/chaincase-app/nolooking/issues/40.

What security issues were you concerned about at that time?

Reply:
Ah, #40  looks different. I was thinking of basically doing dual funding and require that the peer uses `push_amount` but it's impossible to learn enough information in channel acceptor or other events to know whether it's safe to proceed. You'd need to know the exact txid of funding transaction if I remember correctly. Perhaps we could make a PR to add RPC for that.